index.js

```diff
@@ -94,6 +78,38 @@ function init(args) {
   return log(logger);
 }
 
+/**
+ * Initialize logger plugins.
+ * Accepts a comma-delimited list of logging plugins to load and
+ * composes them together to return a single function applying all plugins.
+ *
+ * @return {function}
+ */
+function initPlugins() {
+  const CLAY_LOG_PLUGINS = process.env.CLAY_LOG_PLUGINS || '';
+  const modules = CLAY_LOG_PLUGINS
+    .split(',')
+    .map(module => module.trim())
+    .filter(module => !!module)
+    .filter(module => module[0] != '_') // "_" is used to reserve the private namespace.
+    .map((module) => {
+      try {
+        return require(`./plugins/${module}`);
```

Comment:
Is the intent here that relative paths be defined in `CLAY_LOG_PLUGINS` so we can define them on a per project basis? Example: `CLAY_LOG_PLUGINS=../../utils/clay-log-plugin.js`

Reply:
That's a good question @mrollinsiv.

As configured now, this PR intends `CLAY_LOG_PLUGINS` to be an array of strings representing modules that exist _within_ the `clay/clay-log` project's `plugins` directory.

eg. `CLAY_LOG_PLUGINS=heap,sentry`

But I can see the appeal in allowing plugins to be loaded from an additional fallback location.
The challenge there would be how to define that directory in a safe way (eg. `CLAY_LOG_PLUGINS_PATH`).

Absolute paths aren't ideal because they won't be consistent locally, in Docker, running on a server, etc...
A relative path from `node_modules/clay-log` seems a little suspect as well.

One option would be instructing developers to put something like `process.env.CLAY_LOG_PLUGINS_PATH = `${__dirname}/utils/clay-log-plugins` near the top of their `index.js/app.js` entrypoint (before we require any modules that use `clay-log`).

I'm open to suggestions!

I'm also curious if this method of extending `clay-log` seems useful to anyone else.
The primary reason I put together this PR is to be able to tie into `clay-log` levels for error reporting, but I was thinking it could maybe open a door to log context like a UUID for each request using `continuation-local-storage` or Async Hooks down the road.